src/azure/basic-azure-cli-facade.ts

```diff
@@ -75,7 +78,9 @@ export class BasicAzureCliFacade implements AzureCliFacade {
     );
     this.checkForErrors(result);
 
-    return JSON.parse(result.stdout) as Tag[];
+    log.debug(`listTags: ${JSON.stringify(result)}`);
```

Comment:
Now we still have to add the log statements all the time. But I guess that's okay since we now log all error JSONs

Reply:
The problem is we dont always generate a JSON object from the response. So otherwise we would need to log sometimes and sometimes implicitly log via the `generateJson()` function which is even worse imo... :(